### PR TITLE
Index page creation

### PR DESCRIPTION
**Pull request by kbedell**
Created tests, model, views, routes, and controller for the index page for festivals. Resolved hound errors. (Team voted to require strings with double quotations, instead of single)

Note: Paired: Kat & David

### CODE REVIEW

File: spec/features/user_views_all_festivals_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "users views a list of all festivals" do
+
+    let!(:festival) { Festival.create(name: "Festivus", city: "Boston", country: "US", date: Date.new(2016, 07, 06), user_id: 1) }
+    let!(:festival2) { Festival.create(name: "Festivus2", city: "Boston", country: "US", date: Date.new(2016, 07, 07), user_id: 1) }
```
Comment (davidrf) [inline spec/features/user_views_all_festivals_spec.rb]: I want you to setup FactoryGirl for this project and use that

**Comment by kbedell** [comment]
Updated tests to include the use of factory girl

**Comment by kbedell** [comment]
Changed before statements to let statements in tests

File: spec/features/user_views_all_festivals_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature "users views a list of all festivals" do
+  let!(:festival) { FactoryGirl.create(:festival)}
+  let!(:festival2) { FactoryGirl.create(:festival, name: "Festivus2", date: Date.new(2016, 07, 07))}
+
```
Comment (jennceng) [inline spec/features/user_views_all_festivals_spec.rb]: For the two above lines put a space between the ending parentheses and end }, after the merge that is

**Pull request merged**